apps/api/src/app/feature-flags/feature-flags.module.ts

```diff
@@ -2,12 +2,23 @@ import { Module } from '@nestjs/common';
 
 import { FeatureFlagsController } from './feature-flags.controller';
 
+import { FeatureFlagsService, LaunchDarklyService } from './services';
+
 import { SharedModule } from '../shared/shared.module';
 import { AuthModule } from '../auth/auth.module';
 
+const featureFlagsService = {
+  provide: FeatureFlagsService,
+  useFactory: async () => {
+    return new FeatureFlagsService();
+  },
+};
+
+const providers = [LaunchDarklyService, featureFlagsService];
```

Comment:
I'm wondering whether these two should be added to the `application-generic` package instead

Reply:
Good point, in that way we can also use it inside the `worker`, `ws` and other services

Reply:
I have a fear of `application-generic` becoming a trash can where everything shared ends up there.
I am biased regarding the usage of feature flags in the worker, as it is the lowest level of our system, and I consider feature flags discrimination should be as early as possible in the API layer. On the other side I would understand the need to use feature flags for the WS app. Though one could say that should be controlled in the client (Web app) and to retrieve the logic from the client we have 2 options: to call to our API and have a one place of responsibility or use the Launch Darkly client SDK package.

Reply:
This is how the `application-generic` is used right now 🥲
I feel like it might be useful for all the apps, I can imagine a situation where we want the request to go through the API layer and just feature flag something in the Worker layer, let's say new calculations for the timed digest functionality, for some users we will calculate it with "old logic" and for others using the "new logic" (just the example).

Reply:
maybe let's start like this, and then if we will see the need then we can move to the `application-generic`